cypress/testData/testUser.js

```diff
@@ -0,0 +1,57 @@
+import user1 from "../../src/assets/user/user1.svg";
+import user2 from "../../src/assets/user/user2.svg";
+import Pizza from "../../src/assets/pizza.jpg";
+
```

Comment:
Doesn't this file already exist? I'm confused

Reply:
Sorry might make been pushed by accident